lib/cocoapods/installer/project_cache/target_cache_key.rb

```diff
@@ -167,6 +204,7 @@ def self.from_aggregate_target(sandbox, aggregate_target)
               res.relative_path_from(sandbox.project_path.dirname).to_s
             end
             contents['FILES'] = (relative_resource_file_paths + relative_on_demand_resource_file_paths).sort_by(&:downcase)
+            contents['RESOURCES'] = resource_dependencies.flatten.uniq.sort_by(&:downcase) if resource_dependencies.any?
```

Comment:
I'm open to changing this key since its not immediately obvious how `FILES` and `RESOURCES` are different.